ecs_pod.go

```diff
@@ -29,3 +28,16 @@ func (p *ECSPod) Stop(ctx context.Context) error {
 func (p *ECSPod) Delete(ctx context.Context) error {
 	return errors.New("TODO: implement")
 }
+
+// ECSPodInfo provides information about the current status of the pod.
+type ECSPodInfo struct {
+	// Status is the current status of the pod.
+	Status string
+}
+
+// OwnedSecret is a named secret that may or may not be owned by its pod.
+type OwnedSecret struct {
```

Comment:
Why not just call this something like `PodSecret` or something? Since it may or may not be owned by its pod, `OwnedSecret` is a bit misleading.

Reply:
Naming is hard. Your name sounds better.

Reply:
Yes, naming sucks :) haha